packages/calypso-e2e/src/data-helper.ts

```diff
@@ -32,19 +32,44 @@ export function getCalypsoURL(
 }
 
 /**
- * Returns the credential for a specified account from the configuration file.
+ * Returns the credential for a specified account from the secrets file.
  *
- * @param {string} username Username of the account for which the password is to be obtained.
- * @returns {string} Credential entry found in the configuration file for the given username.
- * @throws {Error} If username does not correspond to a valid entry in the configuration file.
+ * @param {string} accountType Type of the account for which the credentials are to be obtained.
+ * @returns {string[]} Username and password found in the secrets file for the given account type.
+ * @throws {Error} If accountType does not correspond to a valid entry in the file.
  */
-export function getAccountCredential( username: string ): string {
+export function getAccountCredential( accountType: string ): string[] {
 	const testAccounts: { [ key: string ]: string } = config.get( 'testAccounts' );
-	try {
-		return testAccounts[ username ];
-	} catch ( err ) {
-		throw new Error( `Credential for username: ${ username } not found in configuration file.` );
+	if ( ! Object.keys( testAccounts ).includes( accountType ) ) {
+		throw new Error(
+			`Secrets file did not contain credentials for requested user ${ accountType }.`
+		);
 	}
+
+	const [ username, password ] = testAccounts[ accountType ];
+	return [ username, password ];
+}
+
+/**
+ * Returns the site URL for a specified account from the secrets file.
+ *
+ * @param {string} accountType Type of the account for which the site URL is to be obtained.
+ * @returns {string} Site URL for the given username.
+ * @throws {Error} If the accountType does not have a site URL defined, or accountType does not have an entry in the file.
+ * @throws {TypeError} If invalid URL.
+ */
+export function getAccountSiteURL( accountType: string ): string {
+	const testAccounts: { [ key: string ]: string } = config.get( 'testAccounts' );
+	if ( ! Object.keys( testAccounts ).includes( accountType ) ) {
+		throw new Error( `Secrets file did not contain URL for requested user ${ accountType }.` );
+	}
+
+	const [ , , url ] = testAccounts[ accountType ];
+	if ( ! url ) {
+		throw new TypeError( `Secrets entry for ${ accountType } has no site URL defined.` );
```

Comment:
nit: this looks more like a [`ReferenceError`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ReferenceError)
```suggestion
		throw new ReferenceError( `Secrets entry for ${ accountType } has no site URL defined.` );
```